src/Avalonia.Build.Tasks/Avalonia.Build.Tasks.csproj

```diff
@@ -6,7 +6,7 @@
         <BuildOutputTargetFolder>tools</BuildOutputTargetFolder>
         <DefineConstants>$(DefineConstants);BUILDTASK;XAMLX_CECIL_INTERNAL;XAMLX_INTERNAL</DefineConstants>
         <CopyLocalLockFileAssemblies Condition="$(TargetFramework) == 'netstandard2.0'">true</CopyLocalLockFileAssemblies>
-        <NoWarn>NU1605</NoWarn>
+        <NoWarn>NU1605;CS8632</NoWarn>
```

Comment:
Why do we need it?

Reply:
Ah yeah, it's because Avalonia.Build.Tasks doesn't enable nullability but it does include files via links which have nullable annotations.